sdk/src/main/java/pt/ua/dicoogle/sdk/QueryDimInterface.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (C) 2014  Universidade de Aveiro, DETI/IEETA, Bioinformatics Group - http://bioinformatics.ua.pt/
+ *
+ * This file is part of Dicoogle/dicoogle-sdk.
+ *
+ * Dicoogle/dicoogle-sdk is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Dicoogle/dicoogle-sdk is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Dicoogle.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package pt.ua.dicoogle.sdk;
+
+import pt.ua.dicoogle.sdk.QueryInterface;
+import pt.ua.dicoogle.sdk.datastructs.SearchResult;
+import pt.ua.dicoogle.sdk.datastructs.dim.Patient;
+import pt.ua.dicoogle.sdk.datastructs.dim.Serie;
+import pt.ua.dicoogle.sdk.datastructs.dim.Study;
+
+/**
+ * Created by bastiao on 02-02-2017.
+ */
+public interface QueryDimInterface extends QueryInterface {
```

Comment:
I noticed that you haven't yet documented this new API. This is important to fully understand what we're adding and how it cooperates with other plugins.